save-backend/build.gradle.kts

```diff
@@ -19,8 +35,12 @@ tasks.withType<Test> {
 }
 
 dependencies {
+    liquibaseRuntime("org.yaml:snakeyaml:1.15")
```

Comment:
Does it require explicit dependency? Also, you are using only xml, why dependency on yaml?